.github/workflows/CI.yaml

```diff
@@ -51,9 +51,9 @@ jobs:
           cache: gradle
       - name: Frontend component testing
         run: timeout -v 15m ./testing/execute_frontend_component_tests.sh
-# Uncomment the lines below to enable debug logging for investigating flaky component tests.
-#        env:
-#          DEBUG: cypress:*
+        # Uncomment the lines below to enable debug logging for investigating flaky component tests.
+        #        env:
+        #          DEBUG: cypress:*
```

Comment:
Additional indentation can probably be reverted

Reply:
ah yes, this was my auto-linter

Reply:
reverted